trace-mini-agent/src/trace_flusher.rs

```diff
@@ -42,12 +51,25 @@ impl TraceFlusher for ServerlessTraceFlusher {
         if traces.is_empty() {
             return;
         }
+        info!("Flushing {} traces", traces.len());
+
+        debug!("Traces to be flushed: {:?}", traces);
```

Comment:
Another interpolation:

```suggestion
        debug!("Traces to be flushed: {traces:?}");
```